bitnami/kibana/templates/deployment.yaml

```diff
@@ -13,9 +13,13 @@ spec:
     matchLabels: {{- include "common.labels.matchLabels" . | nindent 6 }}
   template:
     metadata:
-      {{- if .Values.podAnnotations }}
-      annotations: {{- include "common.tplvalues.render" (dict "value" .Values.podAnnotations "context" $) | nindent 8 }}
-      {{- end }}
+      annotations:
```

Comment:
If you don't set up podAnnotations and you don't use TLS, I think that annotations will be empty no? Is it works?